dusty/cli/test.py

```diff
@@ -21,16 +22,23 @@
 from docopt import docopt
 
 from ..payload import Payload
-from ..commands.test import (run_app_or_lib_tests, test_info_for_app_or_lib, pull_repos_and_sync_commands,
-                             ensure_valid_suite_name)
+from ..commands.test import (run_app_or_lib_tests, test_info_for_app_or_lib, pull_repos_and_sync,
+                             ensure_valid_suite_name, run_all_app_or_lib_suites)
 
 def main(argv):
     args = docopt(__doc__, argv, options_first=True)
-    if not args['<suite_name>']:
-        return Payload(test_info_for_app_or_lib, args['<app_or_lib_name>'])
-    else:
+    if args['<suite_name>'] == '--all':
```

Comment:
Yeah this is why i wanted you to take a look.  Docopt refuses to distinguish --all option I am trying to provide.  If i put it after gcweb it will always interpret it as <suite_name>. I like the idea of reserving all.